plugins/core/webview/src/q-ui/components/login.vue

```diff
@@ -154,36 +102,14 @@ export default defineComponent({
             } else if (type instanceof ExistConnection) {
                 window.ideApi.postMessage({ command: 'selectConnection', connectionId:  type.pluginConnectionId})
             }
-        }
-    },
-    mounted() {
-        console.log('login mounted')
-        window.changeTheme = this.changeTheme.bind(this)
-        window.ideApi.postMessage({command: 'prepareUi'})
```

Comment:
moved to root.vue
https://github.com/aws/aws-toolkit-jetbrains/pull/4370/files#diff-d831ab0cf73aa597f4bf0c976d638e38dc9abb52042ceea0bc6b6b81439b0bc9R43-R54